README.md

```diff
@@ -98,4 +100,41 @@ We can extend the concept of the `as` syntax by generalizing the right hand side
 import value from "module" as {key1: "value1", key2: [1, 2, 3]};
 ```
 
-This would allow module attributes to scale up to multiple pieces of data, not privileging a single attribute.
+This would allow module attributes to scale to support a variety of metadata. It is currently unclear if the generalized constant form will be pursued in this proposal or in a follow up. In either case the addition of the generalized constant form is not hard to implement on top of the existing [spec text](https://tc39.github.io/proposal-module-attributes/). 
+
+## FAQ
+
+### Why not out of band?
+
+Why not both? The champions of this proposal think that exploring both an in- and out of band solutions to this problem are desirable. Depending on the environment in which JavaScript is being executed in either approach could be seen as desirable.
+
+While an in band solution is more verbose, it is also more straightforward for developers to adopt. For large project with many dependencies developers will not have to worry about creating a large manifest by compiling the meta data of all of their dependencies. Module authors will also not have to worry about shipping manifest in order for consumers to be able to run their modules.
```

Comment:
Could we also mention that it's less verbose for smaller projects developed in a simple way, because they can just write the metadata inline instead of having to create this extra manifest by hand?

Reply:
done